Doc - ssh hosts/nodes (#2715)
vmware-tanzu/community-edition: docs/site/content/docs/latest/tips.md

# Connect to Cluster Nodes with SSH

> You can use SSH to connect to individual nodes of management clusters or workload clusters. The SSH key pair that you created when you deployed the management cluster must be available on the machine on which you run the SSH command. Consequently, you must run `ssh` commands on the machine on which you run `tanzu` commands.

Reviewer: ```suggestion
You can use SSH to connect to nodes in management clusters and workload clusters. The SSH key pair that you created when you deployed the management cluster must be available on the machine on which you run the SSH command.
```

Reviewer: some word smithing and recommending the removal of:

>  Consequently, you must run `ssh` commands on the machine on which you run `tanzu` commands.

This is not required, only the ssh key being on the client (host) you're connecting **from**.

> The SSH keys that you register with the management cluster, and consequently that are used by any workload clusters that you deploy from the management cluster, are associated with the following user accounts:

Reviewer: ```suggestion
The SSH key used to access clusters are associated with the following Linux users:
```

- vSphere management cluster and workload cluster nodes running on both Photon OS and Ubuntu: `capv`
- Amazon EC2 bastion nodes: `ubuntu`
- Amazon EC2 management cluster and workload cluster nodes running on Ubuntu: `ubuntu`
- Amazon EC2 management cluster and workload cluster nodes running on Amazon Linux: `ec2-user`
- Azure management cluster and workload cluster nodes (always Ubuntu): `capi`

> To connect to a node by using SSH, run one of the following commands from the machine that you use as the bootstrap machine:

Reviewer: ```suggestion
To connect to a node by using SSH, run one of the following commands from a machine containing the SSH key:
```

> - vSphere nodes: `ssh capv@<em>node_address`

Reviewer 2: What does the `<em>` mean here? If this is an HTML emphasis tag, that will not be formatted correctly in inside the code formatting markdown.

Should these also includer `-i [path_to_key_file]`?

Reviewer 3: I think we need an update here... based on feedback

Author: removed the html
I'm not sure about adding -i [path_to_key_file], I think the key just needs to be on the machine

- Amazon EC2 bastion nodes, management cluster, and workload nodes on Ubuntu: `ssh ubuntu@<em>node_address`
- Amazon EC2 management cluster and workload nodes running on Amazon Linux: `ssh ec2-user@<em>node_address`
- Azure nodes: `ssh capi@<em>node_address`

> Because the SSH key is present on the system on which you are running the `ssh` command, no password is required.

Reviewer: ```suggestion
> Each cluster host contains the public key. The hosts will not accept an SSH password (`PasswordAuthentication` is set to `no`). 
```
